ipatests/test_integration/test_sssd.py

```diff
@@ -406,6 +406,42 @@ def test_trustdomain_disable_disables_subdomain(self):
         # verify the user can be retrieved after re-enabling trustdomain
         self.master.run_command(['id', user])
 
+    @pytest.mark.xfail(
+        osinfo.id == 'fedora' and osinfo.version_number <= (31,),
+        reason='https://github.com/freeipa/freeipa/pull/4588',
```

Comment:
I'd rather put a link to SSSD ticket in the "reason": https://pagure.io/SSSD/sssd/issue/3721

Reply:
Ok. I'll change.
I've added the GH issue because we have this comment explaining the the xfail: https://github.com/freeipa/freeipa/pull/4588#issuecomment-635376911